### ISSUE TITLE
Im willing to translate this launcher to turkish

### ISSUE THREAD

**Issue by Twig6943** [minecraft-linux/mcpelauncher-manifest#984]
is there a crowdin page? or anything like that?

**Comment by ChristopherHX** [comment]
> is there a crowdin page?

No, there is nothing like that, but we have a lot of translateable text in xml form e.g. here an untranslated file you could translate [mcpelauncher_en.zip](https://github.com/minecraft-linux/mcpelauncher-manifest/files/15366942/mcpelauncher_en.zip) (the file in mcpelauncher-ui-qt doesn't contain all non translated texts)

The ts file can be opened via qtlinguist e.g. https://doc.qt.io/qt-6/qtlinguist-index.html or edited manually in xml

If you translate that sample file you could create a PR to https://github.com/minecraft-linux/mcpelauncher-ui-qt or upload the file here, I could add it then.
At least the dmg macOS and flatpak linux show the translations.
